src/Testing/Http/TestResponseHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Testing\Http;
+
+use PHPUnit\Framework\Assert;
+use Tempest\Http\Response;
+use Tempest\Http\Status;
+
+final readonly class TestResponseHelper
+{
+    public function __construct(private Response $response)
+    {
+    }
+
+    public function getResponse(): Response
```

Comment:
FYI, I added these getters as shorthands. It doesn't make sense that users should access the wrapped response for such simple methods.

Reply:
You're right. Ideally we add methods like `assertBodyContains` etc. 

Let's say I added these getters so that my current tests could succeed, now we can continue to improve our test helpers. (But first let's get this merged)